app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -381,7 +381,7 @@ private fun Content(
 
                             OtherUserProfileTabItem.DEVICES -> {
                                 getOtherUserClients()
-                                OtherUserDevicesScreen(state.otherUserClients)
+                                OtherUserDevicesScreen(state.otherUserClients, state = state)
```

Comment:
Why don't we just pass then `state` and refer to `state.otherUserClients` inside `OtherUserDevicesScreen` directly?